### ISSUE TITLE
Capstone update of next branch broke gapstone

### ISSUE THREAD

**Issue by blacktop** [knightsc/gapstone#5]
```bash
# github.com/knightsc/gapstone
../../../../go/pkg/mod/github.com/knightsc/gapstone@v0.0.0-20191231144527-6fa5afaf11a9/arm64_constants.go:22:11: fatal error: 'capstone/capstone.h' file not found
 #include <capstone/capstone.h>
          ^~~~~~~~~~~~~~~~~~~~~
```

**Comment by blacktop** [comment]
capstone guy released 4.0.2 and rebased his next branch

**Comment by blacktop** [comment]
Hmm this might be a homebrew thing and not a gapstone thing
